### PR TITLE
added missing repositories

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

Added optaweb-employee-rostering and optaweb-vehicle-routing.
Both deploy jobs are now correctly created. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by mbiarnes** [comment]
@yurloc the problem was basically that the optawebs master jobs were removed since we decided only to have the ones for 7.x branch, but after this the decision was to have them for master too.  All the logic was introduced between master and 7.x branches and JDK 1.8 and JDK 11 -  but the two jobs weren't listed and so not created.

File: job-dsls/jobs/deploy_jobs.groovy [hunk 1]
```diff
@@ -72,6 +72,24 @@ def final REPO_CONFIGS = [
...
+                        "container"             : "wildfly",
+                        "integration-tests"     : "true",
+                        "maven.test.failure.ignore" : "true"
+                ]
+        ],
+        "optaweb-vehicle-routing"  : [
+                mvnGoals: "-e -nsu -fae -B -Pwildfly clean deploy com.github.spotbugs:spotbugs-maven-plugin:spotbugs",
+                mvnProps: [
+                        "full"                  : "true",
+                        "container"             : "wildfly",
```
Comment (yurloc) [inline job-dsls/jobs/deploy_jobs.groovy:88]: `-Dcontainer=wildfly` not needed for Optawebs.

File: job-dsls/jobs/deploy_jobs.groovy [hunk 2]
```diff
@@ -72,6 +72,24 @@ def final REPO_CONFIGS = [
...
+                        "integration-tests"     : "true",
+                        "maven.test.failure.ignore" : "true"
+                ]
+        ],
+        "optaweb-vehicle-routing"  : [
+                mvnGoals: "-e -nsu -fae -B -Pwildfly clean deploy com.github.spotbugs:spotbugs-maven-plugin:spotbugs",
```
Comment (yurloc) [inline job-dsls/jobs/deploy_jobs.groovy:85]: `-Pwildfly` can be removed as well.

File: job-dsls/jobs/deploy_jobs.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/deploy_jobs.groovy]: DONE

**Comment by yurloc** [comment]
@rsynek Can you confirm these jobs are needed and do not overlap with something else that we have (for example in the Kogito pipelines)?
- https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/deployedRep/job/optaplanner/
- https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/deployedRep/job/optaweb-employee-rostering/
- https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/deployedRep/job/optaweb-vehicle-routing/

**Comment by rsynek** [comment]
@yurloc @mbiarnes The optaplanner one is nice to have in my opinion, as it deploys snapshots after merging, so that we don't have to wait for a nightly build.

I don't see much benefit from deploying optawebs, though - these are not libraries, but end-user applications. Nobody depends on these. Personally, I would just keep the optaplanner, nothing else.

**Comment by mbiarnes** [comment]
@rsynek no prob - so we can do it s before. But then https://github.com/kiegroup/kie-jenkins-scripts/commit/33e5faf6936609d90598a344f8d86685fe5845eb is not needed any more, since only optaplanner will be available for jdk11. What about the quickstarts? We don't have it as deploy job?

**Comment by rsynek** [comment]
@mbiarnes `optaplanner-quickstarts` even don't deploy to any Nexus at all as they don't produce any binaries somebody could use. It's just the source code that brings value to users.

**Comment by mbiarnes** [comment]
@rsynek @yurloc so we can revert https://github.com/kiegroup/kie-jenkins-scripts/commit/33e5faf6936609d90598a344f8d86685fe5845eb and close this one - we will have only optaplanner as deploy job that will build with JDK 11

**Comment by yurloc** [comment]
Thanks @rsynek for clarification!

I agree, we can revert 33e5faf and close this. That's step 1.

Step 2 will be removing the two optaweb jobs from Jenkins to stop running them uselessly and producing failure notifications. @mbiarnes could you please take care of that? Sorry for the confusion!

**Comment by mbiarnes** [comment]
@yurloc OK - will do this

**Pull request closed without merging**